File: Project Euler #006 - Sum square difference/sum_square_difference.cpp
#include <iostream>

using namespace std;

long long sum_square_difference (long long n) {
    long long s1 = (n*(n+1)/2)*(n*(n+1)/2);
    long long s2 = ((n)*(n+1)*(2*n+1))/6;
    return s1-s2;
}

int main() {
    int t;
    cin >> t;
    for(int a0 = 0; a0 < t; a0++){
        long long n;
        cin >> n;
        cout << sum_square_difference(n) << endl;
    }
    return 0;
}

File: Project Euler #006 - Sum square difference/SumSquareDifference.java
import java.util.*;

public class SumSquareDifference {

    public static long getSumSquareDifference (long n) {
        long s1 = (n*(n+1)/2)*(n*(n+1)/2);
        long s2 = ((n)*(n+1)*(2*n+1))/6;
        return s1-s2;
    }
    
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for(int a0 = 0; a0 < t; a0++){
            long n = in.nextLong();
            System.out.println(getSumSquareDifference(n));
        }
    }
}
